ansible/roles/contiv_network/tasks/main.yml

```diff
@@ -21,33 +33,12 @@
   copy: src=netmaster.service dest=/etc/systemd/system/netmaster.service
   when: run_as == "master"
 
+# resolves an issue with netmaster: https://github.com/contiv/netplugin/issues/210
 - name: start netmaster
   shell: systemctl daemon-reload && systemctl start netmaster
   when: run_as == "master"
+  run_once: true
 
-- name: copy environment file for netplugin
-  copy: src=netplugin dest=/etc/default/netplugin
-
-- name: copy systemd units for netplugin
-  copy: src=netplugin.service dest=/etc/systemd/system/netplugin.service
-
-- name: start netplugin
```

Comment:
Should we reuse existing roles to install/start netplugin? This would be needed upon merge into ansible repo perhaps.